RelatedTechnologies/Sofware Assurance Marketplace (SWAMP) CommonAnalysisResultsFormat (SCARF).mdf

```diff
@@ -0,0 +1,49 @@
+Format Description
+==================
+The Software Assurance Marketplace (SWAMP) effort was established to assist in the process of integrating software assurance processes into a continous assurance process. Software assurance includes many activities; SWAMP has a particular focus on automating and integrating multiple static code analysis (SCA) tools. The Swamp Common Assessment Result Format (SCARF) specifies a common format to hold results produced by any SCA tooling integrated within SWAMP.

```

Comment:
>static code analysis (SCA) [](start = 284, length = 26)

In the SARIF spec, we call them just "static analysis tools". Are you using SCA because that's what the SWAMP/SCARF documentation calls them?